server/api.go

```diff
@@ -23,16 +23,18 @@ type APIServer struct {
 	db *sql.DB
 }
 
-func NewAPIServer(db *sql.DB) APIServer {
-	return APIServer{db: db}
+func NewAPIServer(db *sql.DB) *APIServer {
+	return &APIServer{db: db}
 }
 
-func (a *APIServer) userInfo(ctx *gin.Context, accountID int64) {
+func (a *APIServer) UserInfo(ctx *gin.Context) {
 	var (
 		res goauth.Userinfo
 		ok  bool
 	)
 
+	accountID := ctx.Value(ACCOUNT_ID_KEY).(int64)
```

Comment:
This code is unsafe as if the value is missing or bad type it will crash, wondering if we should do the check and return an unexpected error